### PR TITLE
WEB-Hosting-Mail_fonction_php-SK1429

### PR DESCRIPTION
**Pull request by BrenKt**

### CODE REVIEW

File: pages/web_cloud/web_hosting/mail_function_script_records/guide.fr-fr.md [hunk 1]
```diff
@@ -94,7 +94,21 @@ Si vous recevez correctement le message *The email has been sent successfully!*
...
-### Gérer les états « Désactivé », « Bounce » et « SPAM »
+### Gérer les états « Désactivé », « Bounce » et « SPAM » <a name="block-state"></a>
+
+Dans cette partie vous trouverez le détail de chaque état à l'origine du blocage de votre fonction e-mail.
+
+> [!warning]
+>
+> Avant de détailler chacun de ces états, il est nécessaire de comprendre les points qui peuvent dégrader la réputation de votre nom de domaine ou empêcher la réception de vos e-mails.
+>
+> Vérifier les points suivants en amont:
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/web_hosting/mail_function_script_records/guide.fr-fr.md]:
```suggestion
> Vérifiez les points suivants en amont :
```

File: pages/web_cloud/web_hosting/mail_function_script_records/guide.fr-fr.md [hunk 2]
```diff
@@ -94,7 +94,21 @@ Si vous recevez correctement le message *The email has been sent successfully!*
...
+>
+> - La configuration de [l’enregistrement SPF](/pages/web_cloud/domains/dns_zone_spf) dans la zone DNS du nom de domaine.
+> - La configuration de [l’enregistrement DMARC](/pages/web_cloud/domains/dns_zone_dmarc) dans la zone DNS du nom de domaine, **seulement si le serveur de destination l'exige**.
+> - Vérifier la réputation de l’adresse IP à l’origine de l’envoi ([celle de votre hébergement web](/pages/web_cloud/web_hosting/clusters_and_shared_hosting_ip) dans votre cas), à l'aide d'un outil tel que [MXtoolbox](https://mxtoolbox.com/) ou [Spamhaus](https://check.spamhaus.org/).
+> - L'e-mail ne contient pas d'éléments susceptibles d'être interprétés comme du SPAM. Retrouver une liste non exhaustive de ces éléments dans la partie « [Cas n°3 : Envoi d'e-mails légitimes considérés comme du SPAM](#elements-list-spam)» de ce guide.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/web_hosting/mail_function_script_records/guide.fr-fr.md]:
```suggestion
> - L'e-mail ne contient pas d'éléments susceptibles d'être interprétés comme du SPAM. Retrouvez une liste non exhaustive de ces éléments dans la partie « [Cas n°3 : Envoi d'e-mails légitimes considérés comme du SPAM](#elements-list-spam) » de ce guide.
```

File: pages/web_cloud/web_hosting/mail_function_script_records/guide.fr-fr.md [hunk 3]
```diff
@@ -94,7 +94,21 @@ Si vous recevez correctement le message *The email has been sent successfully!*
...
+> - En cas d'absence de blocage du côté d'OVHcloud et que l'e-mail n'a pas été reçu ou rejeté par le destinataire. Contactez le destinataire afin qu'il vérifie si l'e-mail n'a pas été bloqué au niveau du serveur de réception.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/web_hosting/mail_function_script_records/guide.fr-fr.md]:
```suggestion
> - En cas d'absence de blocage du côté d'OVHcloud et si l'e-mail n'a pas été reçu ou rejeté par le destinataire, contactez le destinataire afin qu'il vérifie si l'e-mail n'a pas été bloqué au niveau du serveur de réception.
```

File: pages/web_cloud/web_hosting/mail_function_script_records/guide.fr-fr.md [hunk 4]
```diff
@@ -94,7 +94,21 @@ Si vous recevez correctement le message *The email has been sent successfully!*
 
 - **Vérifiez la taille totale de votre e-mail** : L'e-mail envoyé ne doit pas dépasser la taille totale de **10 Mo** (encapsulation et en-tête inclus). Le contenu même de votre e-mail ne devra donc pas excéder les **7/8 Mo**.
 
-### Gérer les états « Désactivé », « Bounce » et « SPAM »
+### Gérer les états « Désactivé », « Bounce » et « SPAM » <a name="block-state"></a>
+
+Dans cette partie vous trouverez le détail de chaque état à l'origine du blocage de votre fonction e-mail.
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/mail_function_script_records/guide.fr-fr.md]:
```suggestion
Dans cette partie, retrouvez le détail de chaque état à l'origine du blocage de votre fonction e-mail.
```

**Review by Kazarus4 — APPROVED** [review]

**Comment by OVH-Team-Guides** [comment]
My suggestions are not applied

**Review by OVH-Team-Guides — APPROVED** [review]

**Pull request merged**